packages/angular-test-app/src/index.html

```diff
@@ -8,12 +8,12 @@
 <html lang="en">
   <head>
     <meta charset="utf-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1" />
     <title>AngularTestApp</title>
     <base href="/" />
-    <meta name="viewport" content="width=device-width, initial-scale=1" />
     <link rel="icon" type="image/x-icon" href="favicon.ico" />
   </head>
-  <body>
+  <body class="theme-classic-dark">
```

Comment:
Should not be added here

Reply:
When the themeSwitcher package is utilized within the echarts example, the addition of this class becomes redundant.